components/Links/StyledLink.js

```diff
@@ -0,0 +1,22 @@
+import Link from "next/link";
+import styled, { css } from "styled-components";
+
+const StyledLink = styled(Link)`
+  height: 7rem;
+  width: 80%;
+  background-color: hsla(0, 0%, 4%, 0.64);
+  margin: 1rem auto;
+
+  border-radius: 20%;
+  border: 1px solid black;
+  text-align: center;
+  text-decoration: none;
+  ${({ variant }) => {
+    if (variant === "test") {
+      return css`
+        background-color: black;
+      `;
+    }
+  }}
```

Comment:
if you want to adjust only one property based on a condition and there are only two choices (either the condition is met or not), you can decide for an even shorter approach. [See this as a reference](https://github.com/neuefische/web-bo-22-2/blob/main/sessions/react-styled-components/react-styled-components.md#adapting-based-on-props):

```suggestion
background-color: ${({variant}) => (variant === "test" ? "black" : "hsla(0, 0%, 4%, 0.64)")};
```

Reply:
Thanks, I used this just for a test and I don't use the variant "test" anywhere, so i deleted all of it 🙂